[Jamf Protect Event Collector] Computer Events (#34697)
demisto/content: Packs/JamfProtect/ReleaseNotes/1_1_0.md

#### Integrations

##### Jamf Protect Event Collector

- Updated the Docker image to: *demisto/python3:3.11.9.105369*.
- Added the *computer events* to the event collector.

#### Modeling Rules

##### JamfProtect Modeling Rule

Added a mapping for new source log type - "Computers".

> Added a case sensitivity flag equals to true.
> Added a new fields for event description.
> Added a new fields for operation sub type.

Reviewer: ```suggestion
Added a case sensitivity flag that is equal to true.
Added new fields for the event description.
Added new fields for the operation sub-type.
```

#### Integrations

##### Jamf Protect Event Collector

- Added the *computer events* to the event collector.

> - Updated the Docker image to: *demisto/python3:3.10.14.99865*.

Author: ```suggestion
- Updated the Docker image to: *demisto/python3:3.10.14.99865*.

#### Modeling Rules

##### JamfProtect Modeling Rule

- Added a mapping for new source log type - "Computers"
```

> - Added the *computer events* to the event collector.
> - Added the ***Max computer events per fetch***  integration parameter.

Reviewer 2: ```suggestion
Added the *computer events* to the event collector.
```
Contains the deleted one
